spec/requests/network_routers_spec.rb

```diff
@@ -204,4 +204,30 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  describe 'OPTIONS /api/network_routers' do
+    it 'returns a DDF schema for add when available via OPTIONS' do
+      zone = FactoryBot.create(:zone, :name => "api_zone")
```

Comment:
The zone doesn't seem important to your test, the `ems_network` factory should already take care of it for you.